src/Submission/MapDataModel.jsx

```diff
@@ -144,34 +155,6 @@ class MapDataModel extends React.Component {
     }));
   };
 
-  isInteger = (prop) => {
-    if (
-      this.state.nodeType &&
-      this.props.dictionary[this.state.nodeType] &&
-      this.props.dictionary[this.state.nodeType].properties[prop]
-    ) {
-      return (
-        this.props.dictionary[this.state.nodeType].properties[prop].type ===
-        'integer'
-      );
-    }
-    return false;
-  };
-
-  isNumber = (prop) => {
-    if (
-      this.state.nodeType &&
-      this.props.dictionary[this.state.nodeType] &&
-      this.props.dictionary[this.state.nodeType].properties[prop]
-    ) {
-      return (
-        this.props.dictionary[this.state.nodeType].properties[prop].type ===
-        'number'
-      );
-    }
-    return false;
-  };
-
```

Comment:
Are this not used/been moved to another file that you cancelled them?

Reply:
Thank you for catching this! I forgot to include the refactored `castOption()` method to cover for their use cases. I just pushed the change here: https://github.com/chicagopcdc/data-portal/pull/281/commits/0a2ffed801a84fd451f8cca2e7954b11411b8857.